pages/product-listing/product-listing.css

```diff
@@ -120,6 +115,7 @@ input {
     grid-gap: 1.5rem;
     grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
     margin-top: 0.5rem;
+    margin-top: 1rem;
 }
```

Comment:
Either use gap or margin, I don't think u need both.
GAP would be better as it's a grid

Reply:
Hi Nikhil, the margin is for the space between the nav and the list-items